x/lockup/keeper/lock_test.go

```diff
@@ -45,6 +45,52 @@ func (suite *KeeperTestSuite) TestBeginUnlocking() { // test for all unlockable
 	suite.Require().NotEqual(locks[0].IsUnlocking(), false)
 }
 
+func (suite *KeeperTestSuite) TestBeginForceUnlock() {
```

Comment:
Thanks for the test.

Would it be possible to convert it to be table-driven? [Ref](https://github.com/osmosis-labs/osmosis/blob/4d38caa443d33e561fe99029a3ff6ae0ac57726f/CONTRIBUTING.md#L65-L69) 

Would be great to have separate cases for readability when:
- new lock id is returned if the lock was split
- else same lock id